crates/router/src/connector/paypal.rs

```diff
@@ -172,8 +172,9 @@ impl ConnectorCommon for Paypal {
         let error_reason = response.details.map(|error_details| {
             error_details
                 .iter()
-                .map(|error| format!("description - {} ; ", error.description))
-                .collect::<String>()
+                .map(|error| format!("description - {}", error.description))
+                .collect::<Vec<String>>()
+                .join(", ")
```

Comment:
Let's separate these by semicolons itself; let's not introduce any changes in behavior.